src/main/java/org/sonarlint/intellij/tasks/ServerUpdateTask.java

```diff
@@ -154,7 +154,7 @@ private void updateProjects(ServerConfiguration serverConfiguration, TaskProgres
     for (Map.Entry<String, List<Project>> entry : projectsPerProjectKey.entrySet()) {
       try {
         updateProject(serverConfiguration, entry.getKey(), entry.getValue(), monitor);
-      } catch (Exception e) {
+      } catch (Throwable e) {
```

Comment:
When project is disposed, IntelliJ is raising an AssertionError, which is not an exception... I added a check, but there is still a chance of race condition, so I prefer to catch all here.